packages/core/src/card-model.ts

```diff
@@ -227,20 +227,23 @@ export default class CardModel {
       return;
     }
 
-    for (let field of this.schemaModule.default.loadedFields(newSchemaInstance)) {
-      try {
-        await this.getField(field, newSchemaInstance);
-      } catch (err: any) {
-        let newError = new UnprocessableEntity(
-          `Could not load field '${field}' for ${
-            this.state.type === 'loaded' ? 'card ' + this.url : 'new card of type ' + this.parentCardURL
-          }`
-        );
-        newError.additionalErrors = [err, ...(err.additionalErrors || [])];
-        throw newError;
-      }
-      if (this.recomputePromise !== recomputePromise) {
-        return;
+    let fields = this.schemaModule.default.loadedFields(newSchemaInstance);
+    if (fields.length) {
```

Comment:
i'm unsure why we need this guard. if `fields.length` is 0, then wont the for loop just do nothing?

Reply:
this must be left over from earlier, I'll remove it